src/state/application/reducer.ts

```diff
@@ -43,7 +43,6 @@ export enum ApplicationModal {
   TAX_SERVICE,
   TIME_SELECTOR,
   VOTE,
-  WALLET,
```

Comment:
Is this really necessary?

Reply:
no, but as @cbachmeier pointed out, it's dead code now